src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProjects.cs

```diff
@@ -25,8 +26,19 @@ public GetProjects(IProjectsService projectService)
         public IActionResult Run(
             [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "v1/projects")] HttpRequest req)
         {
-            var projects = this.projectService.GetProjects();
-            return new OkObjectResult(projects);
+            try
+            {
+                var projects = this.projectService.GetProjects();
+                return new OkObjectResult(projects);
+            }
+            catch (ProjectsException e)
+            {
+                return new ContentResult
+                {
```

Comment:
You need to return an object with this format:
{
"message" : "message of error"
}